src/Features/Hud/Watermark.cpp

```diff
@@ -26,11 +28,13 @@ class WatermarkHud : public Hud {
 		int screenWidth, screenHeight;
 		engine->GetScreenSize(nullptr, screenWidth, screenHeight);
 
-		Surface::HFont font = 6;
+		Surface::HFont headerFont = scheme->GetFontByID(33);
+		Surface::HFont subTextFont = scheme->GetFontByID(32);
 
-		int width = surface->GetFontLength(font, "%s", WATERMARK_MSG);
+		int xPos = screenWidth - surface->GetFontLength(subTextFont, "%s", WATERMARK_MSG_HELPTEXT) - 80;
 
-		surface->DrawTxt(font, (screenWidth - width) / 2, screenHeight - 150, Color{255, 255, 255, 100}, "%s", WATERMARK_MSG);
+		surface->DrawTxt(headerFont, xPos, screenHeight - 120, Color{255, 255, 255, 100}, "%s", WATERMARK_MSG_HEADER);
+		surface->DrawTxt(subTextFont, xPos, screenHeight - 70, Color{255, 255, 255, 100}, "%s", WATERMARK_MSG_HELPTEXT);
```

Comment:
```suggestion
		surface->DrawTxt(subTextFont, xPos, screenHeight - 70, Color{255, 255, 255, 100}, WATERMARK_MSG_HELPTEXT);
```
Formatting may not be necessary here.